substratools/algo.py

```diff
@@ -123,7 +122,7 @@ def _train_fake_data(self, *args, **kwargs):
         replaced by the opener fake data.
 
         By default, it only calls directly `Algo.train()` method. Override this
-        method if you want to implement a different behaviour.
+        method if you want to implement a different behavior.
```

Comment:
As a side note, "behaviour" is perfectly valid in the UK.

Reply:
I know, but we use american english throughout the platform. And the "cancelled/canceled" debate is still fresh ;)